public/v4/apps/baikal.yml

```diff
@@ -0,0 +1,26 @@
+captainVersion: 4
+services:
+    '$$cap_appname':
+        image: ckulka/baikal:$$cap_version
+        volumes:
+            - '$$cap_appname-data:/var/www/baikal/Specific'
+            - '$$cap_appname-config:/var/www/baikal/config'
+        caproverExtra:
+            containerHttpPort: '80'
+caproverOneClickApp:
+    variables:
+        - id: '$$cap_version'
+          label: Baikal Version
+          defaultValue: 'nginx'
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/ckulka/baikal/tags
+          validRegex: "/^([^\\s^\\/])+$/"
+    instructions:
+        start: |-
+            Baïkal is a lightweight CalDAV+CardDAV server. It offers an extensive web interface with easy management of users, address books and calendars. It is fast and simple to install and only needs a basic php capable server. The data can be stored in a MySQL or a SQLite database.
+             More details: https://sabre.io/baikal/
+        end: |-
+            Baïkal  has been successfully deployed!
+    displayName: Baïkal (vbbot)
```

Comment:
`(vbbot)` should be removed - unless the name of the app is actually `Baïkal (vbbot)` ?